README.md

```diff
@@ -13,13 +13,14 @@ An enhanced, animated and customizable react-native modal.
 * Listeners for the modal animations ending
 * Resize itself correctly on device rotation
 * Swipeable
+* Scrollable
 
 ## Demo
 
 <p align="center">
 <img src="https://raw.githubusercontent.com/mmazzarolo/react-native-tips/master/imgs/modal.gif" height="300" />
 <img src="https://raw.githubusercontent.com/mmazzarolo/react-native-modal-datetime-picker/master/extras/datetimepicker-ios.gif" height="300" />
-<img src="https://raw.githubusercontent.com/mmazzarolo/react-native-animated-modal/master/extras/example-modal.gif" height="300" />
+<img src="https://raw.githubusercontent.com/pistonsky/react-native-modal/master/extras/example-modal.gif" height="300" />
```

Comment:
Nice!
Do you also have any production app that uses react-native-modal?
I'll happily add a gif of it and a link to the store.

Reply:
Not yet. My team is getting ready to use it, and so I decided to submit this PR beforehand.